### ISSUE TITLE
QA Function Notes

### ISSUE THREAD

**Issue by bleonard314** [IntegralEnvision/integral#26]
I ran the QA function and it worked perfectly. I have a few notes and wish list items below:

- [x] Cell formatting for borders and check boxes in script worksheets (e.g. "main.R") is inconsistent.
- [ ] Add the ability to autofill "QA Instructions" tab from script header.
- [x] Add the ability to include multiple scripts in one QA file.
- [x] "Tag for Findings" hyperlink tied to an absolute path.
- [x] Unclear the purpose of the additional "QA Tags".
- [ ] Integral logo shrunk in script worksheet.

**Comment by jzadra** [comment]
Thanks Ben! These are great, I will work to include them.  FYI The script should already have the ability to include multiple scripts in one QA file.  However, they need to be in the same directory.  Each directory has it's own QA file for scripts.  If this is not desirable, let's discuss.

**Comment by kheal** [comment]
I am not able to get two scripts writing to the same file on Linux.  If I try the file is corrupted.  I suspsect because it is opening in libreoffice but I am not sure.

**Comment by jzadra** [comment]
> I am not able to get two scripts writing to the same file on Linux. If I try the file is corrupted. I suspsect because it is opening in libreoffice but I am not sure.

Does this happen even when the file is not currently open in libreoffice?

**Comment by kheal** [comment]
Yes, I just checked that.  It creates an appropriately named sheet, but it is empty. Good gut check though.
